app/src/main/kotlin/com/mapzen/erasermap/view/MainActivity.kt

```diff
@@ -439,6 +462,18 @@ public class MainActivity : AppCompatActivity(), MainViewController, RouteCallba
         }, 100)
     }
 
+    override fun centerOnTappedFeature(features: List<Feature>, position: Int) {
```

Comment:
Perhaps the `centerOnCurrentFeature(features: List<Feature>)` method above could be refactored to simply call this method passing `pager.getCurrentItem()` as the `position` param?

Maybe also rename this method to `centerOnFeature(...)` to make it more generic?

```kotlin
    override fun centerOnCurrentFeature(features: List<Feature>) {
        val pager = findViewById(R.id.search_results) as SearchResultsView
        centerOnFeature(features, pager.getCurrentItem())
    }
```